docs/cudf/source/api_docs/series.rst

```diff
@@ -214,7 +214,6 @@ Combining / comparing / joining / merging / encoding
 
    Series.append
    Series.update
-   Series.one_hot_encoding
```

Comment:
With this removal, we only have two methods in this section. Specifically, we no longer have "encoding" and we can now match the pandas docs header: https://pandas.pydata.org/docs/reference/series.html#combining-comparing-joining-merging

The header should say "Combining / comparing / joining / merging." Same for `DataFrame`.